packages/cli/README.md

```diff
@@ -127,3 +127,615 @@ jobs:
+  This will significantly reduce the time lighthouse takes to run.
+- If we want to run the script in the background we can execute lighthouse headless with the
+  flag `--chrome-flags="--headless=new"`.  
+  It also is helpful when executing lighthouse in th CI.
+
+The basic implementation of a `RunnerConfig` for the above command looks like this:
+
+```typescript
+// code-pushup.config.ts
+// ...
+import { join } from 'path';
+import { AuditOutputs } from '@code-pushup/models';
+
+function runnerConfig(options: Options): RunnerConfig {
+  const { url, onlyAudits } = options;
+  const outputFile = join(process.cwd(), '.code-pushup', 'lighthouse-report.json');
+  return {
+    // npx lighthouse https://example.com --output=json --outputFile=lighthouse-report.json  --onlyAudits=largest-contentful-paint
+    command: 'npx',
+    args: objectToCliArgs({
+      _: ['lighthouse', url],
+      output: 'json',
+      'output-path': outputFile,
+      onlyAudits,
+    }),
+    outputFile,
+    // implementation follows in the next section
+    outputTransform,
+  };
+}
+
+function outputTransform(output: string): AuditOutputs {
+  // Return dummy audit outputs.
+  // Otherwise the CLI will throw an error as the lighthouse report is not of shape AuditOutputs
+  return [
+    {
+      slug: lcpAudit.slug,
+      score: 0,
+      value: 0,
+    },
+  ] satisfies AuditOutputs;
+}
+```
+
+Now we can execute the CLI with `npx code-pushup collect --no-progress` and see a similar output as the following:
+
+**stdout of basic lighthouse plugin**
+
+```sh
+Code PushUp Report - @code-pushup/core@x.y.z
+
+Chrome Lighthosue audits
+● Largest Contentful Paint                                                0
+
+Categories
+┌──────────┬───────┬────────┐
+│ Category │ Score │ Audits │
+└──────────┴───────┴────────┘
+
+Made with ❤ by code-pushup.dev
+```
+
+#### Implement a `outputTransform` function
+
+As the result of the lighthouse run has a different shape than the required `AuditOutputs` we need to add
+a `outputTransform` and implement the transform form a lighthouse report to audit outputs.
+
+**outputTransform for lighthouse report**
+
+```typescript
+// code-pushup.config.ts
+import { Result } from 'lighthouse';
+
+// ...
+
+function outputTransform(output: string): AuditOutputs {
+  // output is content of `lighthouse-report.json` as string so we have to parse it
+  const lhr = JSON.parse(output as Result);
+  return Object.values(lhr.audits).map(({ id: slug, score, numericValue: value, displayValue, description }) => ({
+    slug,
+    score: score,
+    value: parseInt(value.toString(), 10),
+    displayValue,
+    description,
+  }));
+}
+```
+
+Test the output by running `npx code-pushup collect --no-progress --format=md`.
+The CLI argument `--format=md` will create an additional file containing our created detail information form above.
+
+You should see a newly created file `report.md` crated in the folder `.code-pushup` in your current working directory.
```

Comment:
```suggestion
You should see a newly created file `report.md` created in the folder `.code-pushup` in your current working directory.
```

Reply:
```suggestion
You should see a new file `report.md` created in the folder `.code-pushup` in your current working directory.
```
To avoid repeating `created`?